source/includes/usage-examples/connect-code-examples.cpp

```diff
@@ -0,0 +1,112 @@
+#include <mongocxx/instance.hpp>
+#include <mongocxx/client.hpp>
+#include <mongocxx/uri.hpp>
+#include <bsoncxx/json.hpp>
+#include <mongocxx/exception/exception.hpp>
+
+using bsoncxx::builder::basic::kvp;
+using bsoncxx::builder::basic::make_document;
```

Comment:
```suggestion
#include <mongocxx/client.hpp>
#include <mongocxx/instance.hpp>
#include <mongocxx/uri.hpp>
```

Suggest removing unneeded `#include` and `using` statements.